web/beacon-app/src/components/layout/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +81,37 @@ function SideBar({ className }: SidebarProps) {
     invariant(appState?.orgID, 'orgID is not defined');
   }, [appState?.orgID]);
 
+  // Get the version number from the status response without the release level and number.
+  const statusVersion = status?.version.match(/^v?(\d+\.\d+\.\d+)/);
+
+  useEffect(() => {
+    const storedAppVersion = localStorage.getItem('appVersion');
+
+    if (env !== 'development') {
```

Comment:
This was added to prevent the update notification from always appearing in `dev` as the version number isn't updating.